examples/blueprints/dummygroup.yaml

```diff
@@ -0,0 +1,88 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/4.3.dev1/types.yaml
+  - http://www.getcloudify.org/spec/openstack-plugin/2.0.1/plugin.yaml
```

Comment:
Can we use 2.14.22.

Reply:
getting access denied for http://cloudify.co/spec/openstack-plugin/2.14.22/plugin.yaml